.github/workflows/ctest.yaml

```diff
@@ -0,0 +1,35 @@
+# This is the testing workflow
+
+name: CTest
+
+# Controls when the action will run. Triggers the workflow on push or pull request events
+on: [push, pull_request]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "test"
+  test:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    container: ghcr.io/hsu-hpc/testmamico
+    timeout-minutes: 10
```

Comment:
State the reason for the timeout, otherwise, this might be confusing if, some years into the future this limit is too low and nobody knows why it was here in the first place.

Reply:
Done in 54b00a76f70ede241a0b661d6514a87ae872e88f
Maybe even overdone a bit :smile: